'E501 Line too long': Not showing in-line

I am using the Helix editor with pylsp as the language server and python-lsp-ruff as plugin for ruff. Most diagnostics appear inline in the editor as expected except E501 which is related to line legth. Running ruff directly on the violating file shows the error in the terminal.
```shell
ruff test.py
test.py:3:89: E501 Line too long (100 > 88 characters)
```

The log from pylsp is here: 
```
2023-02-24 12:50:41,955 IST - DEBUG - pylsp.config.config -   pylsp_lint [hook]
      config: <pylsp.config.config.Config object at 0x104791f50>
      workspace: <pylsp.workspace.Workspace object at 0x104830ad0>
      document: file:///Users/rkshthrmsh/ws/sandbox/test.py
      is_saved: True

2023-02-24 12:50:41,956 IST - DEBUG - pylsp.config.config - Got user config from PyCodeStyleConfig: {}
2023-02-24 12:50:41,957 IST - DEBUG - pylsp.config.config - Got project config from PyCodeStyleConfig: {}
2023-02-24 12:50:41,957 IST - DEBUG - pylsp.config.config - With configuration: {'plugins': {'pyflakes': {'enabled': False}, 'pydocstyle': {'enabled': False}, 'rope_rename': {'enabled': False}, 'ruff': {'exclude': None, 'ignore': None, 'select': ['D'], 'lineLength': None, 'perFileIgnores': None, 'executable': 'ruff', 'config': None, 'enabled': True}, 'autopep8': {'enabled': False}, 'flake8': {'enabled': False}, 'preload': {'modules': ['OpenGL', 'PIL', 'array', 'audioop', 'binascii', 'cPickle', 'cStringIO', 'cmath', 'collections', 'datetime', 'errno', 'exceptions', 'gc', 'imageop', 'imp', 'itertools', 'marshal', 'math', 'matplotlib', 'mmap', 'mpmath', 'msvcrt', 'networkx', 'nose', 'nt', 'numpy', 'operator', 'os', 'os.path', 'pandas', 'parser', 'rgbimg', 'scipy', 'signal', 'skimage', 'sklearn', 'statsmodels', 'strop', 'sympy', 'sys', 'thread', 'time', 'wx', 'xxsubtype', 'zipimport', 'zlib']}, 'rope_autoimport': {'enabled': False, 'memory': False}, 'jedi_completion': {'fuzzy': True, 'include_class_objects': True, 'include_function_objects': True}, 'rope_completion': {'enabled': False, 'eager': False}, 'pycodestyle': {'enabled': False}, 'yapf': {'enabled': True}, 'pylint': {'enabled': False, 'args': [], 'executable': None}, 'mccabe': {'enabled': False}}, 'rope': {'extensionModules': ['OpenGL', 'PIL', 'array', 'audioop', 'binascii', 'cPickle', 'cStringIO', 'cmath', 'collections', 'datetime', 'errno', 'exceptions', 'gc', 'imageop', 'imp', 'itertools', 'marshal', 'math', 'matplotlib', 'mmap', 'mpmath', 'msvcrt', 'networkx', 'nose', 'nt', 'numpy', 'operator', 'os', 'os.path', 'pandas', 'parser', 'rgbimg', 'scipy', 'signal', 'skimage', 'sklearn', 'statsmodels', 'strop', 'sympy', 'sys', 'thread', 'time', 'wx', 'xxsubtype', 'zipimport', 'zlib']}}
2023-02-24 12:50:41,958 IST - DEBUG - pylsp_ruff.ruff_lint - Got ruff settings: {'exclude': None, 'ignore': None, 'select': ['D'], 'lineLength': None, 'perFileIgnores': None, 'line_length': 100, 'executable': 'ruff', 'config': None, 'enabled': True}
2023-02-24 12:50:41,958 IST - DEBUG - pylsp_ruff.ruff_lint - Calling ruff with args: ['--quiet', '--format=json', '--no-fix', '--force-exclude', '--stdin-filename', '/Users/rkshthrmsh/ws/sandbox/test.py', '--select=D', '--', '-'] on '/Users/rkshthrmsh/ws/sandbox/test.py'
2023-02-24 12:50:41,971 IST - DEBUG - pylsp.config.config -   finish pylsp_lint --> [[]] [hook]

2023-02-24 12:50:41,972 IST - DEBUG - pylsp_jsonrpc.endpoint - Sending notification: textDocument/publishDiagnostics {'uri': 'file:///Users/rkshthrmsh/ws/sandbox/test.py', 'diagnostics': []}
2023-02-24 12:50:43,477 IST - DEBUG - pylsp_jsonrpc.endpoint - Handling notification from client {'jsonrpc': '2.0', 'method': 'textDocument/didSave', 'params': {'text': '"""Implement CCSDS 123.0-B-2 Compression.\n\nCCSDS 123.0-B-2 specifies a standard for Low-Complexity Lossless and Near-Lossless Multispectral and\nHyper\n"""\n', 'textDocument': {'uri': 'file:///Users/rkshthrmsh/ws/sandbox/test.py'}}}
2023-02-24 12:50:43,480 IST - DEBUG - pylsp.config.config -   pylsp_document_did_save [hook]
      config: <pylsp.config.config.Config object at 0x104791f50>
      workspace: <pylsp.workspace.Workspace object at 0x104830ad0>
      document: file:///Users/rkshthrmsh/ws/sandbox/test.py

2023-02-24 12:50:43,480 IST - DEBUG - pylsp.config.config -   finish pylsp_document_did_save --> [] [hook]

2023-02-24 12:50:43,480 IST - DEBUG - pylsp_jsonrpc.endpoint - Handling request from client {'jsonrpc': '2.0', 'method': 'shutdown', 'params': None, 'id': 1}
2023-02-24 12:50:43,480 IST - DEBUG - pylsp_jsonrpc.endpoint - Got result from synchronous request handler: None
2023-02-24 12:50:43,481 IST - DEBUG - pylsp_jsonrpc.endpoint - Handling notification from client {'jsonrpc': '2.0', 'method': 'exit', 'params': None}
```

--- a/README.md
+++ b/README.md
@@ -40,3 +40,6 @@ the valid configuration keys:
  - `pylsp.plugins.ruff.lineLength`: Set the line-length for length checks.
  - `pylsp.plugins.ruff.perFileIgnores`: File-specific error codes to be ignored.
  - `pylsp.plugins.ruff.select`: List of error codes to enable.
+ - `pylsp.plugins.ruff.extendSelect`: Same as select, but append to existing error codes.
+
+For more information on the configuration visit [Ruff's homepage](https://beta.ruff.rs/docs/configuration/).

--- a/pylsp_ruff/ruff_lint.py
+++ b/pylsp_ruff/ruff_lint.py
@@ -33,6 +33,7 @@ def pylsp_settings():
                 "lineLength": None,
                 "perFileIgnores": None,
                 "select": None,
+                "extendSelect": None,
             },
             "pyflakes": {"enabled": False},
             "flake8": {"enabled": False},
@@ -253,6 +254,7 @@ def load_config(workspace: Workspace, document: Document) -> dict:
             "line-length": None,
             "per-file-ignores": None,
             "select": None,
+            "extend-select": None,
         }
 
     else:
@@ -265,6 +267,7 @@ def load_config(workspace: Workspace, document: Document) -> dict:
             "line-length": _settings.get("lineLength", None),
             "per-file-ignores": _settings.get("perFileIgnores", None),
             "select": _settings.get("select", None),
+            "extend-select": _settings.get("extendSelect", None),
         }
 
     return settings

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -7,7 +7,7 @@ name = "python-lsp-ruff"
 authors = [
   {name = "Julian Hossbach", email = "julian.hossbach@gmx.de"}
 ]
-version = "1.0.5"
+version = "1.1.0"
 description = "Ruff linting plugin for pylsp"
 readme = "README.md"
 requires-python = ">=3.7"

--- a/tests/test_ruff_lint.py
+++ b/tests/test_ruff_lint.py
@@ -77,12 +77,15 @@ def test_ruff_config_param(workspace):
         mock_instance = popen_mock.return_value
         mock_instance.communicate.return_value = [bytes(), bytes()]
         ruff_conf = "/tmp/pyproject.toml"
-        workspace._config.update({"plugins": {"ruff": {"config": ruff_conf}}})
+        workspace._config.update(
+            {"plugins": {"ruff": {"config": ruff_conf, "extendSelect": ["D", "F"]}}}
+        )
         _name, doc = temp_document(DOC, workspace)
         ruff_lint.pylsp_lint(workspace, doc)
         (call_args,) = popen_mock.call_args[0]
         assert "ruff" in call_args
         assert f"--config={ruff_conf}" in call_args
+        assert "--extend-select=D,F" in call_args
 
 
 def test_ruff_executable_param(workspace):
@@ -122,6 +125,7 @@ exclude = [
     "blah/__init__.py",
     "file_2.py"
 ]
+extend-select = ["D"]
 [tool.ruff.per-file-ignores]
 "test_something.py" = ["F401"]
 """
@@ -172,8 +176,10 @@ def f():
     _list = []
     for diag in diags:
         _list.append(diag["code"])
-    # Assert that ignore is working as intended
+    # Assert that ignore and extend-select is working as intended
     assert "E402" in _list
+    assert "D103" in _list
+    assert "D104" in _list
     assert "F841" not in _list
 
     # Excludes

